specification/attributes/amortization_handling.md

```diff
@@ -0,0 +1,34 @@
+# Amortization Handling
+
+Amortization is a process used to break down and spread purchase costs over the period of time or term of use. 
+
+Amortization is primarily used in purchasable commitment-based discounts. When a purchase is applicable to resources, like commitment-based discounts, the amortized cost of a resource takes the initial payment and term into account and distributes it out based on resource usage, attributing the prorated cost for each hour of billing. Amortization provides a method for FinOps practitioners to reallocate significant purchase charges to the appropriate audience in support of their cost allocation efforts. 
```

Comment:
Looks like this comment got flagged as outdated due to other changes. I copied the change to a suggestion I made. I'll close this one since GitHub is hiding it anyway.